### PR TITLE
:sparkles: Add space for long descriptions and add custom label

### PR DESCRIPTION
**Pull request by ibolton336**
- Adds custom label for wizard cards

### CODE REVIEW

File: client/src/app/components/target-card.tsx [hunk 1]
```diff
@@ -137,7 +138,9 @@ export const TargetCard: React.FC<TargetCardProps> = ({
                   </DropdownItem>,
                 ]}
               />
-            ) : null}
+            ) : (
+              readOnly && item.custom && <Label color="grey">Custom</Label>
```
Comment (gildub) [inline client/src/app/components/target-card.tsx:142]: `readOnly` test is not needed, only `item.custom`.

And the true case first would help make it easier to read:
```ts
<FlexItem className={spacing.mlAuto}>
  {readOnly
    ? item.custom && <Label color="grey">Custom</Label>
    : item.custom && (
        <KebabDropdown
          dropdownItems={[
            <DropdownItem key="edit-custom-card" onClick={onEdit}>
              {t("actions.edit")}
            </DropdownItem>,
            <DropdownItem key="delete-custom-card" onClick={onDelete}>
              {t("actions.delete")}
            </DropdownItem>,
          ]}
        />
      )}
</FlexItem>
```

**Review by gildub — COMMENTED** [review]
Looks good, just a reading code issue.

File: client/src/app/components/target-card.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/components/target-card.tsx]: Good idea - Fixed.

**Review by gildub — APPROVED** [review]
LGTM

**Pull request merged**